guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc

```diff
@@ -281,7 +288,7 @@ Review the results and address any highlighted error conditions before performin
 ----
 +
 If the script fails due to missing or outdated packages, you must download and install these separately.
-For more information, see the https://access.redhat.com/documentation/en-us/red_hat_satellite/{AccessRedHatComVersion}/html-single/installing_satellite_server_from_a_disconnected_network/installing-satellite-server-disconnected#resolving-package-dependency-errors_satellite[Resolving Package Dependency Errors] section in the _Installing {ProjectServer} from a Disconnected Network_ guide.
+For more information, see the {InstallingDisconnectedDocURL}resolving-package-dependency-errors_satellite[Resolving Package Dependency Errors] section in the _Installing {ProjectServer} in a Disconnected Network Environment_ guide.
```

Comment:
```suggestion
For more information, see {InstallingDisconnectedDocURL}resolving-package-dependency-errors_satellite[Resolving Package Dependency Errors] in _Installing {ProjectServer} in a Disconnected Network Environment_.
```

Maybe it'd be a good idea to add `InstallingDisconnectedDocTitle` as attribute to `guides/common/attributes-titles.adoc`.

Reply:
Hello! I added the attribute in a separate commit. Do you suggest squashing it to into the first commit or should I leave it as it is?

Reply:
Oh, I think I made an error here: `InstallingServerDisconnectedDocTitle` is already defined in `guides/common/attributes-titles.adoc`. So suggesting `InstallingDisconnectedDocTitle` was wrong on my behalf. Sorry for causing confusion!

It would probably be best to someday move this to "attributes-base.adoc", but that's not something that needs to be done within this PR.

Reply:
Understood. I updated the commits with the correct attribute.